### ISSUE TITLE
Cannot Favorite recipes that have been migrated from mealie_alpha

### ISSUE THREAD

**Issue by gwilliams1205** [mealie-recipes/mealie#1658]
When trying to create favorites within the new beta app, you cannot select any objects that were migrated in. I can select any new items that were scrapped or entered new.

**Comment by hay-kot** [comment]
Hey! There's an issue template that helps gather information about your installation. Please use that in the future.

Can you provide any relevant logs as well as your deployment details?

**Comment by gwilliams1205** [comment]
Apologies, I didn't see the template.

First Check


This is not a feature request

- [x] I added a very descriptive title to this issue.

- [x] I used the GitHub search to find a similar issue and didn't find it.

- [x] I searched the Mealie documentation, with the integrated search.

- [x] I already read the docs and didn't find an answer.

What is the issue you are experiencing?
Cannot favorite recipes that have been migrated from Mealie_Alpha into the new Beta 1.0.0beta-4

**Deployment**
Docker on Ubuntu
**Deployment Details**
**Details**
Version: v1.0.0beta-4
Build:13850cda1f9cb02a8c788030ba6cfa96bba33571 
Application mode: Production
Demo status: Not Demo
API Poort: 9000
API-documentation: enabled
Database type: sqlite
Recipe Scraper Version: 14.11.0
**Checks**
Secure Site: Yes
Server Side Base URL: Yes
LDAP Ready: No
Email Configured: Yes
Docker Volumes: success
